content/pt-br/docs/concepts/storage/volume-health-monitoring.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Monitoramento de Saúde do Volume
+content_type: concept
+weight: 100
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+{{< glossary_tooltip text="CSI" term_id="csi" >}} O monitoramento de saúde de volume CSI permite que os drivers CSI detectem condições anormais de volume dos sistemas de armazenamento subjacentes e os relatem como eventos em {{< glossary_tooltip text="PVCs" term_id="persistent-volume-claim" >}} ou {{< glossary_tooltip text="Pods" term_id="pod" >}}.
+
+<!-- body -->
+
+## Monitoramento de Saúde do Volume
+
+_O monitoramento de saúde do volume do Kubernetes_ é uma parte da forma como o Kubernetes implementa a Interface de Armazenamento de Contêiner (CSI). O recurso de monitoramento de saúde de volume é implementado em dois componentes: um controlador de Monitor Externo de Saúde e o {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}.
```

Comment:
I think "saúde" sounds a bit strange when talking about inanimate things. Perhaps "integridade" is a better fit for the topic in this page?

Reply:
Ok, sounds better, corrected